### PR TITLE
AAI-366: Clean up v2 endpoints

### PR DESCRIPTION
**Pull request by gcs272**
* Add connection details
* Fill in scheduling params
* Misc minor fixes

### CODE REVIEW

File: openapi-v3.yaml [hunk 1]
```diff
@@ -1056,50 +1054,122 @@ components:
...
-          type: "integer"
+          type: integer
+          description: The numeric ID for the connection
         account_id:
-          type: "integer"
-        created_by_id:
-          type: "integer"
-          description: "User ID who created the connection"
+          type: integer
+          description: The numeric ID for the associated account
```
Comment (kaitlinr) [inline openapi-v3.yaml:1065]: Nice - this description is much cleaner!

File: openapi-v3.yaml [hunk 2]
```diff
@@ -1388,31 +1493,36 @@ components:
           format: "date-time"
         owner_thread_id:
           type: "string"
+          nullable: true
         executed_by_thread_id:
           type: "string"
         artifacts_saved:
           type: "boolean"
-        artifacts_s3_path:
+        artifact_s3_path:
```
Comment (kaitlinr) [inline openapi-v3.yaml:1501]: Was this an error in our documentation? I am assuming the actual endpoint uses `artifact_s3_path`?

**Review by kaitlinr — APPROVED** [review]
Thanks Graham for the attention to detail here!

File: openapi-v3.yaml (hunk 2 above)
Comment (eiof) [reply openapi-v3.yaml:1501]: Same question, want to make sure this is intentional!

**Review by eiof — APPROVED** [review]

File: openapi-v3.yaml (hunk 2 above)
Comment (gcs272) [reply openapi-v3.yaml:1501]: yep! that's what the actual endpoint returns

**Pull request merged**